netci.groovy

```diff
@@ -11,6 +11,8 @@ def project = GithubProject
 
         if (osName == 'Windows_NT') {
             runtimes.add('Desktop')
+        } else {
+            runtimes.add('Mono')
```

Comment:
I was going for the quickest route for now, as I thought that I will have to figure out mono install location etc. But we don't need to do that, we can build the mono configuration using .net itself. so, `./cibuild.sh --scope Test --target Mono` should be fine. I will add that tomorrrow and update the PR.